knowledge_base/Contests.md

```diff
@@ -0,0 +1,48 @@
+# Contests
+
+Contests is a new feature, based on contest contracts, available for communities on Common. This document outlines the implementation of Contests on our platform.
+
+A contest contract is an on-chain, repeating game which receives content submissions and votes on submitted content. It then determines the winning content based on relative voting power applied.
+
+Common Contests are an application of contest contracts using Common content. Winners receive a token-denominated prize from a community-designated pool.
+
+## Basic Rules
+
+Each community may have at most one contest running. Contests continue to run even if the prize pool is empty, but no payments will be made to winners in such cases.
+
+Admins cannot interfere with the content submission and voting processes. To be able to change constructor variables, admins must deploy a new contest contract with their new desired variables.
+
+On-chain data submission will be conducted and sponsored by Commonwealth Labs. As of 240509, Common takes a 10% fee from contest distributions.
+
+Contests logic is managed by `Contest.sol`, and financial processes by the `FeeManager.sol` contract.
+
+## Contest Funding and Economics
+
+Contest prizes are primarily funded through the namespace’s Community Stake transaction commissions. This will be used by the `FeeManager.sol` contract, and CommunityStake fees generated from the community's staking activities will be directed to the contest contract. Additionally, direct transfers to the contest contract are also supported.
```

Comment:
We can consider to update this part as follows:

Contest prizes are primarily funded through the namespace’s Community Stake transaction commissions. These transaction commissions will be collected by the `FeeManager.sol` contract, and CommunityStake fees generated from the community's staking activities will be directed to the contest contract based on the share of the community stake in the `FeeManager.sol` contract. Additionally, direct transfers to the contest contract are also supported.